frontend/integration-tests/tests/kubevirt/vm.wizard.scenario.ts

```diff
@@ -1,89 +1,47 @@
 /* eslint-disable no-undef, max-nested-callbacks */
-
 import { execSync } from 'child_process';
-import { browser, by, ExpectedConditions as until } from 'protractor';
 import { OrderedMap } from 'immutable';
+import { browser, ExpectedConditions as until } from 'protractor';
 
+// eslint-disable-next-line no-unused-vars
+import { networkResource, provision as provisionOptions, removeLeakedResources, storageResource, VM_BOOTUP_TIMEOUT } from './utils';
 import { appHost, testName } from '../../protractor.conf';
-import { resourceRowsPresent, filterForName, deleteRow, createItemButton, isLoaded, errorMessage } from '../../views/crud.view';
-import { removeLeakedResources } from './utils';
-import { testNAD } from './mocks';
-import * as vmView from '../../views/kubevirt/vm.view';
-
-const VM_BOOTUP_TIMEOUT = 90000;
-const PAGE_LOAD_TIMEOUT = 5000;
-const VM_WIZARD_LOAD_TIMEOUT = 10000;
+import { deleteRow, errorMessage, filterForName, isLoaded, resourceRowsPresent } from '../../views/crud.view';
+import { glusterfsDisk, hddDisk, networkInterface, testNAD } from './mocks';
+import * as wizardView from '../../views/kubevirt/wizard.view';
+import Wizard from './models/wizard';
 
 describe('Kubevirt create VM using wizard', () => {
   const leakedResources = new Set<string>();
```

Comment:
How about move these constants to mock.ts, so other tests can use them as well.

Reply:
Yep, done.